added google home alarm documentation

eliseomartelli opened the pull request:
**Description:**
Added the Google Home Alarms sensor documentation.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#20645
## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

eliseomartelli committed:
added google home alarm documentation

eliseomartelli committed:
fixed date

dshokouhi commented on source/_components/sensor.google_home_alarm.markdown:
```diff
@@ -0,0 +1,63 @@
+---
+layout: page
+title: "Google Home Alarm"
+description: "Instructions on how to integrate a BH1750 light sensor into Home Assistant."
```
May want to update the description as this is not a light sensor :)

eliseomartelli replied:
took care of that 😎

eliseomartelli replied:
Netlify preview ready :)

dshokouhi commented:
Asides from the minor issue, this looks good to me.

dshokouhi reviewed (changes requested):
The description should be updated for the integration.

eliseomartelli committed:
updated documentation for gh alarm sensor

frenck commented on source/_components/sensor.google_home_alarm.markdown:
```diff
@@ -0,0 +1,46 @@
+# Example configuration.yaml entry
+sensor:
+  - platform: google_home_alarm
+    monitored_conditions:
```
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

frenck commented on source/_components/sensor.google_home_alarm.markdown:
```diff
@@ -0,0 +1,46 @@
+monitored_conditions:
+  description: The conditions to monitor.
+  required: false
+  type: map
```
```suggestion
  type: list
```

frenck reviewed (changes requested):
Thanks for the PR @eliseomartelli!

I've reviewed it, and left some comments. Could you please take a look? 👍

eliseomartelli committed:
fixed configuration to reflect the component

eliseomartelli commented:
@frenck I've fixed the configuration to reflect the code review from the component's pull request.

frenck reviewed (dismissed):
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

YogoGit committed:
Correct URL link (#8117)

* Correct URL link

The markdown syntax for the link was reversed.

* Fix ocomment

DubhAd committed:
We didn't document the services (#8147)

Adding documentation on how to use the services. Actually, adding documentation on _what_ the service is ;)

casperklein committed:
'igd' added to ignore listing (#8142)

Bodge-IT committed:
Small changes to help fill a couple of small gaps. (#8111)

* Small changes to help fill a couple of small gaps.

Stopped short of offering a full example such as Harmony Remote config.

* Fix comments

cogneato committed:
Removed reference to guest account (#8099)

Guest access no longer exists. Removed reference to it as an option.

larsvinc committed:
Added spot clean in accordance with the pull request for that functionality: (#8093)

https://github.com/home-assistant/home-assistant/pull/19857

Br4veSt4rr committed:
Update light.yeelight.markdown (#8049)

* Update light.yeelight.markdown

Add extra configuration detail to highlight the existence of auto discovery, see "Duplication of Yeelight in 0.63 #12337"

This will make it easier to add the product.

* Minor changes

fabaff committed:
Lupus (#8156)

* Add supported units (fixes #8154)

* Update logo

adrum committed:
Add support for HomeKit Controller covers (#8096)

fabaff committed:
Update for 0.85

patatman committed:
Adding my config to the Cookbook (#8159)

rohankapoorcom committed:
Documentation update for splitting gpslogger into a component with a device tracker platform (#8152)

* Documentation update for splitting gpslogger into a component with a device tracker platform

* Remove redundant line from the device_tracker

* Fix stale copy/paste

nickrout committed:
Update plum_lightpad.markdown (#8163)

I don't think the project should be an advertising shill for a product. 

Also the "review" is from the manufacturer, so hardly unbiased.

bachya committed:
Add new info about OpenUV API key usage (#8127)

* ness_alarm: Address config error (#8126)

* Add new info about OpenUV API key usage

* Updates for service calls

fabaff committed:
Add defaults and other fixes

fabaff committed:
Update formatting

LouisMT committed:
Fix wrong update interval for SolarEdge sensor (#8167)

choss committed:
fixed a typo (#8168)

soberstadt committed:
Adding documentation for Roku hub and remote (#6944)

* Adding documentation for Roku hub and remote

* revert date change

* Fix year

* PR feedback fixes

iainsmacleod committed:
Update hue.markdown (#8136)

Had the inverse description for allow_hue_groups

tradiuz committed:
Updated to reflect the new API limits (#8124)

The free tier of OpenUV is now limited to 50, not 500.  All existing accounts will drop to that limit in February, but all new accounts have that limit.  The current scan_interval of 2/hr puts it at 48 calls/day (without any reboots triggering extras).  Should we adjust the default to be 1/hr or give documentation on how to not query it until the sun is above the horizon?

exxamalte committed:
Additional information for Geolocation platforms (#8113)

* fixed wrong type

* added missing source to the attributes table

* added state attributes table

* radius default corrected to float value

SNoof85 committed:
Added supported router (#8070)

kennedyshead committed:
Make asuswrt sensor optional (#8032)

* Only sensor setup is changed

* Adding sensorrs config to component

kellerza committed:
Update sensor.sma.markdown (#8050)

basbl committed:
Changed how packages are exposed by the postnl sensor. (#7445)

pszafer committed:
Fix - Mqqt position topic docs more clear (#7562)

* Changes position_topic and state_topic info to be more clear for users.
Also deleted get_position_topic expression as it wasn't clear enough.
Using only position_topic expression from now.

* :pencil2: Tweak

magic-madrigal committed:
Update google_assistant.markdown (#8074)

The documentation is out of date.

klaasnicolaas committed:
Update Intent_script component configuration (#7154)

* Update Intent_script component configuration

* :pencil2: Update file

* :pencil2: Tweaks

* :pencil2: Tweak

* :pencil2: Tweak

Sbaa1 committed:
Update commandline.markdown (#8135)

* Update commandline.markdown

* :pencil2: Tweak

frenck committed:
:pencil2: Spelling, grammar, styling and fixes (#8173)

hAmpzter committed:
Added example to notify-group platform (#7889)

* Added example to notify-group platform

* Removed {% raw %}-tags

* :pencil2: Tweak

* :pencil2: Tweak

Commit:
Update light.mqtt.markdown (#8000)

* Update light.mqtt.markdown

* Update template docs links

* Fix light mqtt links

insertjokehere committed:
Clarify Lifx documentation (#8036)

* Make it clearer that multiple `server`/`broadcast` blocks are supported
* Note that `broadcast` can be set to the unicast address of a bulb

infamousjoeg committed:
Added easy Docker CE install to alt install (#8162)

frenck committed:
:pencil2: Spelling, grammar, styling and fixes (#8178)

emlove committed:
Split time_pattern triggers from time trigger (#8080)

* Split interval triggers from time trigger

* Default smaller interval units to zero

* Rename interval to schedule

* Rename schedule trigger to time_pattern

DubhAd committed:
Adding link to the LLAT section of the docs (#8148)

* Adding link to the LLAT section of the docs

Adding link to the LLAT section of the docs, and to the API docs. This should hopefully make it clearer to folks that this isn't a drop in replacement.

* :pencil2: Made link relative

nkleven committed:
Update (#8183)

Mac Addresses are colon delimited, not dash.

frenck committed:
:pencil2: Spelling, grammar, styling and fixes (#8181)

frenck committed:
:ambulance: Fixes broken device_class links (#8179)

quazzie committed:
Add dismiss (#8121)

rohankapoorcom committed:
Add support for multiple ZoneMinder installations (#8129)

keslerm committed:
Update alarm_control_panel.arlo.markdown (#8191)

Anonym-tsk committed:
Ready (#8175)

Gamester17 committed:
Update zha.markdown to list ConBee and RaspBee via the zigpy-deconz library (#8189)

Update ZHA component markdown to list ConBee and RaspBee via the zigpy-deconz library from @damarco (for zigpy from @rcloran ) which just had its first release.

https://github.com/zigpy/zigpy-deconz

rohankapoorcom committed:
Update GPSLogger docs for the webhook component integration (#8186)

* Update GPSLogger docs for the webhook component integration

* #8186: Add GPSLogger logo to device tracker platform

* #8186 reset permissions

* Swap references from GET to POST

rohankapoorcom committed:
Update documentation for locative with the new webhook setup (#8160)

olskar committed:
correct mistake (#8192)

cvwillegen committed:
Update sensor.imap_email_content.markdown (#8190)

Added documentation for added 'folder' config entry

frenck committed:
:pencil2: Spelling, grammar, styling and fixes (#8187)

balloob committed:
Merge branch 'current' into next

balloob committed:
Add initial draft for 86

balloob committed:
add link original pr

balloob committed:
Add link to interview with Zac

rdkayman committed:
Update vacuum.xiaomi_miio.markdown (#8198)

* Update vacuum.xiaomi_miio.markdown

token retrievable in log file :-)

* :pencil2: Tweak

yeralin committed:
Update homekit.markdown (#8195)

* Update homekit.markdown

Discovered new pairing hangs cause and resolution

* Fix numbering

cvwillegen committed:
Added links for web addresses (#8199)

* Added links for web addresses

* :pencil2: Tweak

* :pencil2: Tweak

txNgineer committed:
Update trigger markdown to expand twilight discussion and sunrise timing (#8194)

* Update trigger markdown to expand twilight discussion and sunrise timing

These changes expand on the sun angle for addition darkness by including detail on the three twilight definitions. A brief discussion has also been added to the time trigger section to explain why many automations that are relative to sunrise often fail.  Since sunrise is a different day than any time before midnight, using timestamps is a reasonable option.

* Minor changes

* :pencil2: Tweaks

finity69x2 committed:
Update automation.markdown (#8196)

* Update automation.markdown

* :pencil2: Tweaks

Chaotic committed:
Update calendar.google.markdown (#8204)

Add in the expected format for the offset to help prevent bugs being entered that it only supports 2 digits (it does but it supports hours)
https://github.com/home-assistant/home-assistant/issues/20168
https://github.com/home-assistant/home-assistant/issues/11886
https://github.com/home-assistant/home-assistant/issues/5791

frenck committed:
:art: Optmizes image assets (#8203)

pvizeli committed:
Update HassOS 2.6

frenck committed:
:hammer: Replaces all hardlinks with relative ones (#8207)

frenck committed:
:shirt: Replaces truthy YAML boolean values (#8212)

frenck committed:
:ambulance: Fixes Liquid warning (#8210)

syssi committed:
Add Xiaomi Airpurifier Pro V7 support and a missing Airfresh paragraph (#8169)

* Add Xiaomi Airpurifier Pro V7 support and a missing Airfresh paragraph

* Add table of supported devices

* Use official product names

* Remove redundancy

tjorim committed:
Remove outdated docs (#8215)

It's supported since 0.84

frenck committed:
:arrow_up: Upgrades nokogiri to 1.10.1 (#8209)

fabaff committed:
Update option

alengwenus committed:
Updated documentation for relay ports for LCN light platform (#7977)

rytilahti committed:
update ha_iot_class for songpal (#8221)

Songpal defaults to local push since https://github.com/home-assistant/home-assistant/pull/19129 .

zhujisheng committed:
Create image_processing.qrcode.markdown (#8208)

* Create image_processing.qrcode.markdown

for the new platform

* Minor changes

tjorim committed:
Fix service call list (#8219)

This was intended as a list but was all on a single line

koreth committed:
Document Lutron RadioRA2 button events (#8164)

* Add Lutron binary sensor

* Fix typo in date

* Buttons now fire events instead of showing up as binary sensors

* Fix grammar error

* Rename "event" attribute to "action"

foxel committed:
Added Xiaomi AirPurifier 2S profile (#8230)

joncar committed:
Initial documentation for new EverLights light component (#8076)

* Everlights light component documentation.

* Review feedback and fix typo.

* Add EverLights logo. Image copyright EverLights. Use on this page approved via email.

* English...

* :pencil2: Tweaks

* :pencil2: Tweaks

frenck commented:
Not stale, parent PR is still open.

robmarkcole committed:
Update sensor.uk_transport.markdown (#8231)

fabaff committed:
Make title linkable

aapjeisbaas committed:
Merge lines were not cleaned up after merge (#8227)

thomasloven committed:
Fix link to ha-style.ts (#8226)

alengwenus committed:
Add documentation for LCN switch platform (#8223)

* Add documentation for LCN switch platform

* Changed ha_release to 0.87

tjorim committed:
Try to make liquid more strict (#8216)

Instead of warnings it should output error messages on the console for each error and stop the build.

Gamester17 committed:
Update zha.markdown with deconz radio type as optional configuration variable (#8211)

Added `deconz` to CONFIGURATION VARIABLES as additional radio type in zha.markdown.

rohankapoorcom committed:
Add docs for the ZoneMinder binary sensor (#8206)

rohankapoorcom committed:
Add docs for verify_ssl configuration added to rest_command and switch.rest (#8205)

mwegner committed:
Documentation for Exposing IMAP Search String (#8034)

* Documentation for Exposing IMAP Search String

To accompany a PR to sensor/imap.py

* Rewrote Last Sentence for More Clarity

emontnemery committed:
Update binary_sensor.command_line.markdown (#8119)

* Update binary_sensor.command_line.markdown

* Update source/_components/binary_sensor.command_line.markdown

Co-Authored-By: emontnemery <erik@montnemery.com>

* Update binary_sensor.command_line.markdown

mitchellrj committed:
Correctly map Nest hvac_state to Home Assistant states. (#8114)

* Correctly map Nest hvac_state to Home Assistant states.

* Avoid the Oxford comma, the standards say

mitchellrj committed:
Correctly map Nest hvac_state to Home Assistant states. (#8114)

* Correctly map Nest hvac_state to Home Assistant states.

* Avoid the Oxford comma, the standards say

frenck committed:
:pencil2: Spelling, grammar, styling and fixes (#8237)

cdce8p committed:
Extend disable Auto Start - HomeKit (#8005)

* Extend disable Auto Start - HomeKit

* Include option with wait_template

* Only use wait_template

towerhand committed:
Update ios.markdown (#8182)

* Update ios.markdown

* Update ios.markdown

* :pencil2: Tweaks

jkeljo committed:
Update greeneye_monitor docs with breaking change. (#7976)

* Update greeneye_monitor docs with breaking change.

* Update to keep pace with changes in the main PR

jkeljo committed:
Update greeneye_monitor docs with breaking change. (#7976)

* Update greeneye_monitor docs with breaking change.

* Update to keep pace with changes in the main PR

adrum committed:
Add changes for alarm_control_panel.homekit_controller (#7965)

* added documenation changes for alarm_control_panel.homekit_controller

* alphabetical order for homekitcontroller_alarm_control_panel change

* bump version for homekit controller alarm control panel

adrum committed:
Add changes for alarm_control_panel.homekit_controller (#7965)

* added documenation changes for alarm_control_panel.homekit_controller

* alphabetical order for homekitcontroller_alarm_control_panel change

* bump version for homekit controller alarm control panel

LouisMT committed:
Update documentation for friendly names in SolarEdge sensor (#8193)

worm-ee committed:
HomematicIP Cloud add cover devices platform (#8170)

* Add HMIP cover devices platform

* Fix ha_category

* :arrow_up: ha_release 0.87

krygal committed:
Documentation for ee_brightbox device tracker (#7962)

* Documentation for ee_brightbox device tracker

* updated release to 0.86

* :arrow_up: ha_release 0.87

quazzie committed:
Mqtt discovery of vacuums (#7910)

* Mqtt discovery of vacuums

See https://github.com/home-assistant/home-assistant/pull/19478

* :pencil2: Tweak

quazzie committed:
Mqtt discovery of vacuums (#7910)

* Mqtt discovery of vacuums

See https://github.com/home-assistant/home-assistant/pull/19478

* :pencil2: Tweak

frenck committed:
:pencil2: Spelling, grammar, styling and fixes (#8239)

awkwardDuck committed:
Update/Remove GTFS feeds. (#8241)

Update the GTFS sensor markdown to remove obsolete/dead links for GBrail.info and PTV Australia. Update the Switzerland link for 2019.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

matkor committed:
Ecoal boiler (#7707)

* Added components.ecoal_boiler docs.

* Sensors and pumps control for ecoal_boiler.

* Follow configuration layout changes made during component PR review.

* Update 'ha_release'

* Update 'ha_release'

* Delete platform docs

* Remove platform docs

* Add redirects

* Various fixes

balloob committed:
Update blog posts

iantrich committed:
edits

iantrich committed:
Update 2019-01-23-lovelace-released.markdown

arigilder committed:
Add additional documentation for new Jewish Calendar sensors (#7838)

* Update docs for new Jewish calendar sensors

* Add havdalah example

* Update docs for three-day holidays

* Attempt to rectify exception while generating docs

* Consolidate description into single line.

* Fix review comments

iantrich committed:
Update 2019-01-23-release-86.markdown (#8246)

reefab committed:
Add path option to octoprint component documentation (#8236)

Add path option to octoprint component documentation

JonasPed committed:
Add documentation for Danfoss Air HRV. (#8180)

* Add documentation for Danfoss Air HRV.

* Change category and add logo.

* Add documentation for Danfoss Air binary sensor and sensor
platform.

* Add brief description of the (binary) sensors supported by danfoss air.

balloob committed:
Remove demo

cgarwood committed:
Update ha_release for danfoss (#8249)

balloob committed:
Update blog posts

balloob committed:
Add breaking changes

balloob committed:
updates

syphernl committed:
Add docs for data parameter in alert component (#6968)

* Add docs for data parameter in alert component

* Fix docs based on feedback

* Alter docs

* Re-add suggestion

* Update docs with proper data parameter

* Add mention of templateable title

balloob committed:
Typos and order change

balloob committed:
Add video

balloob committed:
Add marius lastname

balloob committed:
Add instructions on how to use old UI

balloob committed:
Merge branch 'rc' into current

balloob committed:
Merge branch 'current' into next

balloob committed:
Fix raw tag

balloob committed:
Merge branch 'current' into next

balloob committed:
Fixes

cgtobi committed:
Fix typo time instead of name (#8252)

balloob committed:
Update podcast link

balloob committed:
Release 86.1

bastshoes committed:
Updating radio_type description for Deconz (#8255)

Add `deconz` to `radio_type` description.

Qu3uk committed:
Updated Automation Example with time_pattern (#8256)

I'm 99% sure this needs to be time_pattern now as opposed to time.

cliffordwhansen committed:
GPSLogger new settings (#8254)

* Updated for new webhook settings

Updated the config items to support the new webhooks settings

* Updated image to current version of application

zewelor committed:
Add info about new yeelight start_flow service (#8166)

balloob committed:
blog post Nest (#8264)

* Post about Nest

* Update source/_posts/2019-01-24-nest-cannot-access-data.markdown

Co-Authored-By: balloob <paulus@home-assistant.io>

* Update source/_posts/2019-01-24-nest-cannot-access-data.markdown

Co-Authored-By: balloob <paulus@home-assistant.io>

balloob committed:
Update nest

labrunning committed:
Yaml formatting on input_select example (#8262)

Added extra space before Paulus and Anne Therese

frenck committed:
:pencil2: Spelling, grammar, styling and fixes (#8260)

balloob committed:
Update nest

fabaff committed:
Remove config file stuff (fixes #8251) (#8266)

frenck committed:
Refactor of documentation method for components (2019-0016) (#8265)

* :tractor: Refactor of documentation method for components

* :shirt: Marks Ikea Tradfri ha_config_flow

* :shirt: Fixes additionalwhitespace generation

bforbird committed:
Explain how to get the device running when running hass in docker (#8258)

* Explain how to get the device running when running hass in docker

* Minor changes

fabaff committed:
Fix Sense docs (#8267)

* Remove platform documentation

* Minor changes

fabaff committed:
Merge branch 'current' into next

balloob committed:
Remove warning from Nest docs

fabaff committed:
Update for 0.86

Qu3uk committed:
time_pattern automation update. (#8259)

* time_pattern automation update.

* Update send_a_reminder.markdown

* Update send_a_reminder.markdown

frenck committed:
:ambulance: Do not show empty categories (#8268)

frenck committed:
:tractor: Merges/Redirect MyChevy component pages (#8269)

e-krebs committed:
update nginx_proxy.markdown (#8253)

* update nginx_proxy.markdown

missing information in what to remove from the http section of the configuration.yaml file

* Update nginx_proxy.markdown

frenck committed:
:tractor: Merges/Redirect Telldus Live component pages (#8270)

arsaboo committed:
Update time_pattern docs (#8263)

* Cleanup time_pattern

* Update sensor.tibber.markdown

klaasnicolaas committed:
🚜 Merges/Redirect Velux component pages (#8271)

klaasnicolaas committed:
🚜 Merges/Redirect Zha component pages (#8272)

klaasnicolaas committed:
🚜 Merges/Redirect LiteJet component pages (#8273)

Commit:
Remove 'experimental' for lovelace references (#8276)

* Remove Experimental

Changed to match current wording of info panel on 0.86.0

* Remove Experimental

Lovelace is no longer 'experimental after 0.86.0.

klaasnicolaas committed:
🚜 Merges/Redirect Coinbase component pages (#8274)

balloob committed:
Release 0.86.2

klaasnicolaas committed:
🚜 Merges/Redirect Sleepiq component pages (#8277)

klaasnicolaas committed:
🚜 Merges/Redirect Ecobee component pages (#8275)

* 🚜 Merges/Redirect Ecobee component pages

* :pencil2: Tweaks

Jpsy committed:
Create automation_knx.markdown (#8228)

* Create automation_knx.markdown

The KNX information for HA are rather rudimentary and there are more open questions than answers. 
Even if I am a beginner, I think it is a good idea to share my findings.
This knowledge will grow fast and I will keep this overview updated.

* Basic KNX automation example added

* :pencil2: Tweaks

* :pencil2: Tweaks

amelchio committed:
Add LIFX listen port advanced configuration (#8235)

foxel committed:
Added time throttle filter docs (#8245)

jmtatsch committed:
Add encoding parameter to mqtt automation  (#8247)

* Add encoding parameter to mqtt automation #20292

This is needed to trigger on received images/other byte payloads via mqtt because otherwise the decode fails. See also 
#16004

* Update trigger.markdown

clarify

DubhAd committed:
Fixing critical typo (#8280)

It's not `webook` - there's an **H** missing there ;)

balloob committed:
Update 2019-01-23-release-86.markdown

klaasnicolaas committed:
🚜 Merges/Redirect Tahoma component pages (#8281)

klaasnicolaas committed:
🚜 Merges/Redirect Maxcube component pages (#8282)

klaasnicolaas committed:
🚜 Merges/Redirect Raspihats component pages (#8283)

bajansen committed:
Update device_tracker.snmp.markdown (#8285)

Added baseoid for Aerohive AP230 and rearranged table of devices alphabetically.

DubhAd committed:
Updating example, adding link (#8290)

Updated the example in the multiple triggers section to be valid. Added link from the top of the page.

jwater7 committed:
Update sensor.scrape.markdown (#8288)

fix a 404

Censored3 committed:
addon/git_pull: enhance with repeat_ignore, corrections, re-ordering (#8257)

* addon/git_pull: enhance with repeat_ignore, corrections, re-ordering

* addon/git_pull: restart_ignore: remove trailing slash requirement and some corrections

dgomes committed:
Utility meter (#8025)

* add utility_meter

* updates

* :pencil2: Tweaks

* spellcheck

* made it into an embedded component

* sensor -> component

* bump version

jon102034050 committed:
Update lutron.markdown (#8294)

lutron component was recently updated to support seeTouch keypads

balloob committed:
Add section on migration

balloob committed:
Update 2019-01-23-lovelace-released.markdown

balloob committed:
Remove tips & tricks

balloob committed:
More LL cleanup

danielsjf committed:
Co2Signal platform (#8037)

* Add CO2Signal documentation

* Add logo from website

* Update sensor.co2signal.markdown

Modifications after review

* :pencil2: Tweak

* Update sensor.co2signal.markdown

Update after review

balloob committed:
0.86.3

rohankapoorcom committed:
Add documentation for splitting up the dovado sensor (#8296)

Jc2k committed:
Remove homekit controller out-of-date warning (#8307)

Remove homekit controller out-of-date  warning

DubhAd committed:
Fixing the example (#8313)

The example isn't valid 🤦‍♂️

hfurubotten committed:
Docs for nilu air_quality platform (#7993)

* Docs for nilu air_pollution platform

* Updated with new location of pollution index

* Fixed stations to be plural

* platform rename from air_pollutants to air_quality
Managed to remove a commit from maintainer.
Edits should be within this renaming commit.

* Update 'ha_release'

nollbit committed:
Improve Tellstick documentation (#8310)

The configuration format for Tellstick can be a bit confusing, but the official documentation is pretty good, so let's link to it.

phithor committed:
Update media_player.nad.markdown for newbie friendlyness. (#8298)

* Update media_player.nad.markdown

Added additional information in the wiki to make it more newbie friendly.

* :pencil2: Tweak

klaasnicolaas committed:
🔨 Remove SSL from example (#8317)

bachya committed:
Add docs for upcoming Lyft sensor changes (#8297)

klaasnicolaas committed:
🚜 Merges/Redirect Lightwave component pages (#8318)

* 🚜 Merges/Redirect Lightwave component pages

* :pencil2: Add Hub back to the HA category

eliseomartelli committed:
Add Iliad Italy Documentation (#7981)

* added iliad docs

* :pencil2: Tweaks

* smaller logo

* Update 'ha_release'

emontnemery committed:
Update binary_sensor.mqtt.markdown (#8309)

* Update binary_sensor.mqtt.markdown

* Update binary_sensor.mqtt.markdown

emontnemery committed:
Update example for JSON attributes topic (#8299)

emontnemery committed:
Document json_attributes_topic for MQTT Alarm (#8300)

* Document json_attributes_topic

* Update alarm_control_panel.mqtt.markdown

* Update alarm_control_panel.mqtt.markdown

emontnemery committed:
Document json_attributes_topic for MQTT Lock (#8308)

* Document json_attributes_topic for MQTT Lock

* Update lock.mqtt.markdown

emontnemery committed:
Document json_attributes_topic for MQTT Vacuum (#8306)

* Document json_attributes_topic for MQTT Vacuum

* Update vacuum.mqtt.markdown

emontnemery committed:
Document json_attributes_topic for MQTT Switch (#8305)

* Document json_attributes_topic

* Update switch.mqtt.markdown

* Update switch.mqtt.markdown

emontnemery committed:
Document json_attributes_topic for MQTT Light (#8304)

* Document json_attributes_topic

* Update light.mqtt.markdown

* Update light.mqtt.markdown

emontnemery committed:
Document json_attributes_topic for MQTT Climate (#8303)

* Document json_attributes_topic

* Update climate.mqtt.markdown

* Update climate.mqtt.markdown

emontnemery committed:
Document json_attributes_topic for MQTT Fan (#8302)

* Document json_attributes_topic

* Update fan.mqtt.markdown

* Update fan.mqtt.markdown

theaquarium committed:
Cover position_topic type should be string, not integer (#8314)

DubhAd committed:
Note about time trigger, and tidy up (#8292)

Removed the out of context block about the sun, under the time pattern. It doesn't appear to fit anywhere, and appeared in the RC merge.

Added warning about `00` and so on

emontnemery committed:
Document json_attributes_topic for MQTT Cover (#8301)

* Document json_attributes_topic

* Update cover.mqtt.markdown

* Update cover.mqtt.markdown

klaasnicolaas committed:
🚜 Merges/Redirect EnOcean component pages (#8319)

knottyau75 committed:
Fix Variable for Device Name (#8311)

klaasnicolaas committed:
🚜 Merges/Redirect Fritzbox component pages (#8322)

klaasnicolaas committed:
🚜 Merges/Redirect Lutron component pages (#8323)

klaasnicolaas committed:
🚜 Merges/Redirect Comfoconnect component pages (#8325)

klaasnicolaas committed:
🚜 Merges/Redirect ISY994 component pages (#8326)

klaasnicolaas committed:
🚜 Merges/Redirect Lutron Caseta component pages (#8324)

fabaff committed:
Merge branch 'current' into next

klaasnicolaas committed:
🚜 Merges/Redirect EDP_Redy component pages (#8327)

klaasnicolaas committed:
🚜 Merges/Redirect Neato component pages (#8328)

klaasnicolaas committed:
🚜 Merges/Redirect Neato component pages (#8328)

fabaff committed:
Merge branch 'current' into next

skion committed:
Needs libffi-dev to compile on Debian 9.7. (#8330)

klaasnicolaas committed:
🚜 Merges/Redirect Homematic component pages (#8331)

klaasnicolaas committed:
🚜 Merges/Redirect Homekit controller component pages (#8333)

balloob committed:
Version bump to 0.86.4

bachya committed:
Add doc updates for upcoming Ambient PWS changes (#8244)

* Add doc updates for upcoming Ambient PWS changes

* Updated method to Cloud Push

aerialls committed:
Add documentation for Synology SRM device tracker (#8240)

* Add documentation for Synology SRM device tracker

* :pencil2: Tweaks

dgomes committed:
Adds Integration Sensor (#8012)

* energy sensor

* address review comments

* from energy meter -> integration meter

* :pencil2: Tweaks

* bump version

stealthhacker committed:
Add Recollect Waste platform documentation (#8171)

* Added Recollect Waste platform documentation.

* Using configuration tags now.

* Fixed invalid formatting of configuration section.

* Fixed invalid formatting of configuration section.

* Adjusted scan_interval description as platform has been updated to use built-in scan_interval.

* :pencil2: Tweaks

klaasnicolaas committed:
🚜 Merges/Redirect Joaoapps_join controller component pages (#8336)

klaasnicolaas committed:
🚜 Merges/Redirect BeagleBone Black GPIO component pages (#8337)

klaasnicolaas committed:
🚜 Merges/Redirect Raspberry Pi GPIO component pages (#8339)

klaasnicolaas committed:
🚜 Merges/Redirect WaterFurnace component pages (#8340)

klaasnicolaas committed:
🚜 Merges/Redirect OpenUV component pages (#8341)

pvizeli committed:
Update beta to RC

balloob committed:
Fix typo

Thanks to @cgtobi

tonymitchell committed:
Update dhcp_software option in docs to match code (#8348)

The code uses 'odhcpd' while the document listed the option as 'dhcpd'. Fixing documentation

balloob committed:
Merge branch 'current' into next

andrewsayre committed:
New component SmartThings and switch platform (#8184)

* Added SmartThings component and switch platform documentation.

* Minor changes per style guidelines

* Embedded switch platform.

* :pencil2: Tweaks

GidoHakvoort committed:
Added documentation for ROVA sensor (#7780)

* Added documentation for ROVA sensor

* Update source/_components/sensor.rova.markdown

Fixed example configuration

Co-Authored-By: GidoHakvoort <gido@hakvoort.it>

* Update sensor.rova.markdown

* Update sensor.rova.markdown

* Update sensor.rova.markdown

* :pencil2: Tweak

* Update sensor.rova.markdown

Update ha_release

choss committed:
adding documentation for fritz.dect 100 sensor component (#8242)

* adding documentation for new sensor component

* :pencil2: Tweak

* :pencil2: Add HA Release

andrewsayre committed:
Add ha_version for SmartThings component and switch (#8355)

klaasnicolaas committed:
🚜 Merges/Redirect SmartThings component pages (#8357)

zumitnl committed:
Add precision parameter to dsmr sensor (#8106)

eliseomartelli commented:
Waiting https://github.com/home-assistant/home-assistant.io/pull/8346 to update accordingly.

ludeeus committed:
Change documentation for googlehome (#8346)

* Initial attempt for googlehome component documentation

* Corrected typo in "platform"

* Rename file

* Move keys to configuration section

* Added missing description:

* Add quotes

* revert  44464ed2

scop committed:
Add Huawei LTE Notify docs (#7934)

* Add Huawei LTE Notify docs

* Use recipient instead of target in config

* Bump HA release to 0.88

rohankapoorcom committed:
Fix platform links when using embedded components (#8379)

andrewsayre committed:
Add SmartThings Light platform (#8365)

* Add SmartThings Light platform docs

* Minor tweaks

* Specify the classic ST mobile app

* Updated ha_version

eliseomartelli committed:
added sensor googlehome

eliseomartelli committed:
updated documentation

eliseomartelli committed:
Merge branch 'google-home-alarm-sensor' of https://github.com/eliseomartelli/home-assistant.io into google-home-alarm-sensor

eliseomartelli committed:
removed sensor page
